pkg/speccheck/speccheck_test.go

```diff
@@ -4,6 +4,8 @@ import (
 	"fmt"
 	"testing"
 
+	"encoding/json"
```

Comment:
Nit: would be good to put it together with other imports. Feel free to merge with or without this change.